code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -136,6 +136,19 @@
 			AT.fire_act(temperature, volume)
 	return
 
+/obj/effect/hotspot/proc/reaction_volume_calculation(datum/gas_mixture/gasmix)
```

Comment:
Using a proc for this is less then stellar, also needs profiling

Reply:
![image](https://user-images.githubusercontent.com/54709710/143678611-881b44ad-f7c0-4b5c-a40f-a48a9b92aa72.png)

Reply:
😢